example/eblob_backend.c

```diff
@@ -1078,6 +1080,10 @@ static int blob_send(struct eblob_backend_config *cfg, void *state, struct dnet_
 		if (err)
 			goto err_out_send_fail_reply;
 
+		/* stop if an error is occurred during write */
+		if (ctl->write_error)
+			goto err_out_put;
+
```

Comment:
That's a very different commands.
This code changes how server-send command works, it prevents sending more keys if some of them have failed.

Code in `dnet_server_send_write` works in server-send **iterator** which stopped iterator if there are write errors. This code does not replace what you have removed.